test.sh

```diff
@@ -35,8 +35,7 @@ pushd orbit
 popd
 
 # Reset the tests and mail directories
-sudo rm -rf test email/logs/* email/mail/*
-mkdir -p test
```

Comment:
we should leave the directory creation in here in case it doesn't exist

Reply:
good point. Will fix